### ISSUE TITLE
Question: Support for different themes for diff. users

### ISSUE THREAD

**Issue by falcon646** [namazso/SecureUxTheme#21]
No sure If this is an issue or was never intented as a function. Hence asking as a question. Does it support themes per user ?
I tried applyig a different theme for another local user on my system but it changes the theme for other users as well.

**Comment by namazso** [comment]
This seems to work on my machine: https://streamable.com/4d2geo

**Comment by falcon646** [comment]
Yeah, it seems to work when there is one stock windows theme is involved. But while aplying diferent "custom" themes for different users. What I mean is if I apply a custom theme for user1 and then go ahead and apply another custom theme for user2 and try to log in back to user1 , It will have the custom theme I applied for user2.

**Comment by falcon646** [comment]
Update : Everything is working great. I am able to add custom theme to other user without affecting the current one. What I was doung wrong was I was uning the themetool from the secureuxtheme installer as I didnt want to install virtual studio and other stuffs on my vm to build the entire thing. By change ,I stumbled upon the stand alone themetool.exe from another issue and used that to apply theme for other users. Works without any issue.

Thanks for this creating this tool. This is exactly what I wanted, to apply themes without breaking windows on updates. Cheers mate. Great job.

**Comment by namazso** [comment]
> What I was doing wrong was I was using the themetool from the secureuxtheme installer

Hm, i wonder why it didn't work from the installer.
Did you maybe happen to not be administrator on one of the accounts and thus running as different user to comply with the admin requirement of the installer?

**Comment by falcon646** [comment]
> Did you maybe happen to not be administrator on one of the accounts and thus running as different user to comply with the admin requirement of the installer?

Yup. Thats excatly what I did the first time. Installed on the admin acount, applied the theme. Switched to the standard user account. It had the same theme I applied for admin. Ran the exe here but did not install. Clicked on themetool, it asked for admin credentials, gave the credentials and applied another theme. And that changed the theme for admin as well.
